ofnetAgent.go

```diff
@@ -517,3 +580,498 @@ func (self *OfnetAgent) DummyRpc(arg *string, ret *bool) error {
 	log.Infof("Received dummy route RPC call")
 	return nil
 }
+
+/*
+Bgp serve routine does the following:
+1) Creates inb01 router port
+2) Add MyBgp endpoint
+3) Kicks off routines to monitor route updates and peer state
+*/
+func (self *OfnetAgent) Serve() error {
+	time.Sleep(5 * time.Second)
+	self.WaitForSwitchConnection()
+
+	self.modRibCh = make(chan *api.Path, 16)
+	self.advPathCh = make(chan *api.Path, 16)
+
+	timeout := grpc.WithTimeout(time.Second)
+	conn, err := grpc.Dial("127.0.0.1:8080", timeout, grpc.WithBlock(), grpc.WithInsecure())
```

Comment:
Can we use a different port for gobgp? 8080 is a very common port and we'll likely have port conflicts.